galera/Project1/Developer.h

```diff
@@ -10,6 +10,8 @@ class Developer : public Employee
     string _language;
 
 public:
-    void setLanguage(string language);
+    enum class  language { c, c_sharp, python, c_plusplus, java };
+    void setLanguage(language language);
     string getLanguage() const;
```

Comment:
Your interface should be consistent. If you have `void setLanguage(language language);`, then you also need to have `language getLanguage() const;`